interfaces/Portalicious/src/app/pages/project/project-registrations/components/custom-message-preview/custom-message-preview.component.ts

```diff
@@ -19,21 +25,51 @@ import { SendMessageData } from '~/domains/registration/registration.api.service
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class CustomMessagePreviewComponent {
-  sendMessageData = input<SendMessageData>();
-  // TODO: AB#30847 add additional property for dummy data to be used in the preview
+  readonly projectId = input.required<number>();
+  readonly previewRegistration = input.required<Registration>();
+  readonly messageText = input.required<string>();
```

Comment:
I changed this to accept `messageText` (the body of the message) instead of `sendMessageData` (body of the message if custom, otherwise template key) because I already had the body of the message to pass on in both scenarios.

However, I'm not sure what will be more convenient for the status change popups. If it's gonna be more convenient to pass in the template key for the scenario of templated messages (because maybe we'll only have easy access to that, and not the body of the message?) then I can change this.